main/webapp/modules/core/scripts/util/filter-lists-search.js

```diff
@@ -0,0 +1,68 @@
+/*
+* Derived from Portfolio filterable.js by Joel Sutherland
+*
+* Copyright (C) 2009 Joel Sutherland.
+* Liscenced under the MIT liscense
+*
+* Modified for OpenRefine by Simone Povoscania
+* Modified for OpenRefine by Lucero Garcia
+*/
+
+(function($) {
```

Comment:
something is causing a crash on FF in here...I'm debugging more...but basically fast typing and backspacing are causing it. Are we filling and emptying an array of the characters I type too quickly perhaps?  I'll see...maybe a 500ms delay is needed?

Reply:
```
Script terminated by timeout at:
each@http://127.0.0.1:3333/index-bundle.js:371:12
each@http://127.0.0.1:3333/index-bundle.js:139:17
$.fn.filterListSearch/</<@http://127.0.0.1:3333/index-bundle.js:34540:29
each@http://127.0.0.1:3333/index-bundle.js:372:19
each@http://127.0.0.1:3333/index-bundle.js:139:17
$.fn.filterListSearch/<@http://127.0.0.1:3333/index-bundle.js:34535:30
dispatch@http://127.0.0.1:3333/index-bundle.js:5228:27
add/elemData.handle@http://127.0.0.1:3333/index-bundle.js:4880:28
trigger@http://127.0.0.1:3333/index-bundle.js:5132:12
jQuery.event.trigger@http://127.0.0.1:3333/index-bundle.js:11647:22
trigger/<@http://127.0.0.1:3333/index-bundle.js:5862:17
each@http://127.0.0.1:3333/index-bundle.js:372:19
each@http://127.0.0.1:3333/index-bundle.js:139:17
trigger@http://127.0.0.1:3333/index-bundle.js:5861:15
$.fn.filterListSearch@http://127.0.0.1:3333/index-bundle.js:34531:26
Refine.OpenProjectUI.prototype._searchInput/<@http://127.0.0.1:3333/index-bundle.js:35325:28
dispatch@http://127.0.0.1:3333/index-bundle.js:5228:27
add/elemData.handle@http://127.0.0.1:3333/index-bundle.js:4880:28
```

Reply:
Nope, there's no delay needed... it's got hundreds of secs delay after I type 5 or 6 chars and then grinds to a halt on performance when I'm recording and looking at the performance in FF using Web Developer tools.  Really slowwwwww.  I'm wondering what is causing it?

I'll leave it to you to debug this more.

Reply:
Workin on it, I let you know, thanks for the feedback

Reply:
It had an stack overflow beacuse of the keyup detection I decided to limited to an `Enter`to start the search, let me know what you think